WireUI/Sources/WireSettingsUI/Resources/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,28 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+"settings.exportBackup.action" = "Back Up Now";
+"settings.exportBackup.description" = "Create a backup to preserve your conversation history. You can use this to restore history if you lose your computer or switch to a new one. The backup file is not protected by Wire end-to-end encryption, so store it in a safe place.";
+"settings.restoreFromBackup.action" = "Restore from Backup";
+"settings.restoreFromBackup.description" = "The existing history on this device remains and will be completed by the new backup. You can restore history from all your devices and different platforms but not from another account.";
+"exportBackup.title" = "Set password";
+"exportBackup.description" = "The backup will be compressed and encrypted with a password. Make sure to store it in a secure place.";
+"exportBackup.button" = "Back Up Now";
+"setBackupPassword.title" = "Password (OPTIONAL)";
```

Comment:
What about using the `exportBackup` prefix here as well?
So that on Crowding it is clear that those strings are related?

Reply:
changed